plugins/acr/package.json

```diff
@@ -45,6 +45,7 @@
     "@backstage/dev-utils": "1.0.31",
     "@backstage/test-utils": "1.5.4",
     "@janus-idp/cli": "1.8.6",
+    "@redhat-developer/red-hat-developer-hub-theme": "*",
```

Comment:
@jerolimov wouldnt' this pick any matched version, why not pin the version ?

Reply:
@karthikjeeyar , good question. I pinned one in `packages/app/package.json`. Pinning different per plugins doesn't work nicely and I don't wanted that we need to touch all packages.json for each theme update. With this we can just update `packages/app` and the `yarn.lock` of course.

Reply:
Okay, but I dont see this package.json packages/app in this PR or in main branch, am I missing something? 🤔

Reply:
You're right, I missed it, maybe it's dropped in a rebase conflict. It's there again.